### DISCUSSION TITLE
follow.js capsolver?

### DISCUSSION THREAD

**Discussion by ExamV1** [masterking32/twitch-account-creator#32] [Q&A]
Why is the CapSolver(captcha) needed for the follow.js and why are proxies not optional? I understand for the account creator

**Comment by masterking32** [comment]
This project is based on Twitch-Account-Creator and the follow.js is just an example.
If you know any other way to follow without proxy or CapSolver you can tell me.
So If you don't know and you want to use it, So you need to follow my way with all requirements.

**Reply by ExamV1** [reply]
yes you made one, I sent it in the discord general chat containing your file, I removed the proxies being needed but with or without the proxies the follows still send but they send slow or only send a few

**Reply by masterking32** [reply]
so if you don't use a proxy maybe your account marked as a bot and gets banned. That's your's of what you exactly want to do.

**Reply by ExamV1** [reply]
> so if you don't use a proxy maybe your account marked as a bot and gets banned. That's your's of what you exactly want to do.

Yes that is it, but instead, you could have it do the proxies are optional so you can use proxies or do not want to use proxies, many reasons to not use proxies, is that proxies could not be working, you may not want to pay for proxies, and if using free proxies the bot to send follows will more than likely send much slower, but you can simply use no proxies and have it send follows from the chosen tokens where there isn't a rate limit, and account will get banned eventually if used a lot for botting purposes anyway.
